SofaModulePostProcessorShareManager should detect null parameter

SofaModulePostProcessorShareManager should detect null parameter

```java
Caused by: java.lang.NullPointerException
  at com.alipay.sofa.isle.spring.share.SofaModulePostProcessorShareManager.unableToShare(SofaModulePostProcessorShareManager.java:48)
```

--- a/sofa-boot-project/sofa-boot-core/isle-sofa-boot/src/main/java/com/alipay/sofa/isle/spring/share/SofaModulePostProcessorShareManager.java
+++ b/sofa-boot-project/sofa-boot-core/isle-sofa-boot/src/main/java/com/alipay/sofa/isle/spring/share/SofaModulePostProcessorShareManager.java
@@ -44,8 +44,9 @@ public class SofaModulePostProcessorShareManager {
     }
 
     public boolean unableToShare(Class<?> cls) {
-        return this.filterClassList.contains(cls)
-               || cls.isAnnotationPresent(UnshareSofaModulePostProcessor.class);
+        return cls != null
+               && (this.filterClassList.contains(cls) || cls
+                   .isAnnotationPresent(UnshareSofaModulePostProcessor.class));
     }
 
     public boolean unableToShare(String beanName) {

